newspaper/nlp.py

```diff
@@ -9,17 +9,34 @@
 
 import re
 import math
+from os import path
 
 from collections import Counter
 
 from . import settings
 
-with open(settings.NLP_STOPWORDS_EN, 'r') as f:
-    stopwords = set([w.strip() for w in f.readlines()])
-
 ideal = 20.0
 
+stopwords = set()
 
+def load_stopwords(language):
+    """ 
+    Loads language-specific stopwords for keyword selection
+    """
+    global stopwords
+    
+    # stopwords for nlp in English are not the regular stopwords
+    # to pass the tests
+    # can be changed with the tests
+    if language == 'en':
+        stopwordsFile = settings.NLP_STOPWORDS_EN
+    else:
+        stopwordsFile = path.join(settings.STOPWORDS_DIR,\
```

Comment:
nit: `stopwordsFile` should be `stopwords_file` per python syntax, but letting this fly since your change is crucial.